### PR TITLE
Add portfolio desktop

### PR DESCRIPTION
**Pull request by bimbolabuari**
# Portfolio Microverse Project  🚀 

 > This PR adds a project about building a portfolio with  HTML and CSS that demonstrates the knowledge of Semantic HTML, flexbox, background properties, and mobile-first responsiveness, Responsive web design, CSS media query.

## Built With 🙂 

- Semantic HTML & CSS
- Linters (Stylelint, Web hint, Lighthouse)

## Live Preview  🎉 

[Live Preview Link](https://deploy-preview-4--bimbola-mv-portfolio.netlify.app/)

### CODE REVIEW

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @bimbolabuari  

#### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:blush::blush:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [x] Kindly align the text in the about me section with the image as shown below.
- [x] The text beside the from should be equal to the form height.

## Original Design
right

## Your project
wrong1

wrong2



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Review by Shaher-11 — COMMENTED** [review]
## _Status:_ Invalid:x:
## Hello @bimbolabuari 
This Code Review Request will be marked as invalid in your Dashboard because the linter checks are not passed.
Please make the necessary changes and submit another code review request once you are ready! :smiley:

File: styles.css [hunk 1]
```diff
@@ -526,3 +533,177 @@
...
+    color: #fff;
+  }
+
+  .img-works {
+    height: auto;
+    width: 60%;
+  }
+}
+
+@media only screen and (min-width: 970px) {
```
Comment (henatan99) [inline styles.css]: - [x] OPTIONAL: This min-width value should serve minimum width for desktop version and better to make it in the range 992px-1100px. Yours is a bit smaller than standard desktop widths.

File: styles.css [hunk 2]
```diff
@@ -526,3 +533,177 @@
...
+  }
+
+  .contact-button {
+    align-self: start;
+    font-size: 18px;
+  }
+
+  .greetings-container {
+    margin-left: 6%;
+  }
```
Comment (henatan99) [inline styles.css]: - [x] OPTIONAL: The greetings element seems to take not more than in the original design, so I suggest you give a width of about 50% for desktop view. Kindly, compare yours and the original design from the attached screenshot images.
image   
image

File: index.html [hunk 3]
```diff
@@ -72,6 +78,7 @@ <h3 class="title flex">I'am a Software Developer</h3>
       <h2 class="work-header">My recent Works</h2>
```
Comment (henatan99) [inline index.html:78]: - [ ] OPTIONAL: This header is aligned left and also followed by a horizontal line to the end in the original design. Kindly compare yours and the original design from the screenshot images below. 
image    
image

File: index.html [hunk 4]
```diff
@@ -72,6 +78,7 @@ <h3 class="title flex">I'am a Software Developer</h3>
...
       <div class="works flex">
         <div class="img-works"></div>
+        <div class="work-container">
```
Comment (henatan99) [inline index.html:81]: - [x]  OPTIONAL: The width proportions of the `img-works` and `work-container` classes inside the works element is not as the original design. Please see the attached images for comparison. 
- [x] OPTIONAL: Color differences of some element is also there. 
- [x] OPTIONAL: Same is true for the `btn-flex` class child elements. 
image   
image

**Review by henatan99 — APPROVED** [review]
Hi Abimbola, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- Please take note of the OPTIONAL comments because they are really important even though they are OPTIONAL and not needed for this project's submission. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bimbolabuari** [comment]
@henatan99, Thanks for approving my project 🎉, I have made all the optional changes 🙂

**Pull request merged**